tool/src/massive/munit/command/RunCommand.hx

```diff
@@ -26,15 +26,17 @@
  * or implied, of Massive Interactive.
  */
 package massive.munit.command;
+
 import haxe.ds.StringMap;
 import haxe.io.Eof;
 import massive.haxe.log.Log;
 import massive.haxe.util.RegExpUtil;
-import massive.munit.ServerMain;
 import massive.munit.client.HTTPClient;
+import massive.munit.ServerMain;
 import massive.munit.util.MathUtil;
 import massive.sys.io.File;
 import massive.sys.io.FileSys;
+import neko.vm.Module;
```

Comment:
remove unused

Reply:
this one is used line 275 -> Module.local().name